sktime/forecasting/model_selection/_split.py

```diff
@@ -1051,12 +1051,33 @@ def get_n_splits(self, y: Optional[ACCEPTED_Y_TYPES] = None) -> int:
         n_splits : int
             The number of splits.
         """
+        from sktime.datatypes import check_is_scitype, convert_to
+
         if y is None:
```

Comment:
hm, this is an odd one. `y` has a default, ut that default always throws this error.

Also, only some splitters require `y`. Perhaps we should deal with this with a tag that tells us whether?

Out of scope for this PR though.